package.json

```diff
@@ -59,7 +59,11 @@
     "dist"
   ],
   "types": "dist/index.d.ts",
-  "dependencies": {},
+  "dependencies": {
+    "@types/js-cookie": "^3.0.2",
```

Comment:
Can we move the @types to `devDependencies`? It is only used for TS compilation but not required at runtime and won't be exported by our package so it should go in `devDependencies` to make the bundle size smaller.

```
yarn remove @types/js-cookie
yarn add --dev @types/js-cookie
```